### ISSUE TITLE
Implement Pretraining Rating LinReg

### ISSUE THREAD

**Issue by Bonifatius94** [Bonifatius94/ChessAI.Py#7]
**Suggestions:**
- make the pretraining on GM games data work
- don't try to achieve too much at one:
  - reduce the training dataset to a minimum
  - reduce the model to a minimum
- normalize input features, don't mix up piece bitboards of both sides into one channel with +1/-1 as it struggles with the ReLU activation
 
**Goal:**
- MSE lower than 0.001 (if possible)
- this would equal a deviation of about 0.01

**Comment by Bonifatius94** [comment]
- updated the rating pretraining in #3 
- the results are beter now, but not ideal
- main improvements come from reducing the dataset to very common draws only (more than 20, 30 or even more analyzed games with exactly this position), simplifying the model and properly adjusting the training hyper-parameters
- a remaining issue could be that the model does not know which side is currently drawing, but the position could occur for either side drawing -> there may be score confusions (drawing in a position could be good for one side, but not for the other one)
